app/views/api_docs/vendor_api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,9 @@
+## v1.5 — 16th December 2024
```

Comment:
```suggestion
## v1.6 — 16th December 2024
```

Reply:
Apologies for the confusion - this PR was in draft when you reviewed it, and I added the "do not review" label to avoid wasting anyone's time. I was using the PR to run the CI tests. The release notes will be updated when we officially release version 1.6.

It's ready for review now 👍🏼